src/config/dynamic-theme-fixes.config

```diff
@@ -21688,6 +21688,12 @@ INVERT
 
 sciencedirect.com
 
+INVERT
```

Comment:
As it works fine in Firefox then I can't merge inverts because it will fix Chrome based browsers but break Firefox.

I managed to get what is needed from Firefox.

Instead of INVERTs add this lines on the end of CSS section:
```
.article-textbox {
    background-color: ${rgb(213, 211, 207)} !important;
}
.Article .topic-link {
    color: ${rgb(46, 50, 52)} !important;
    text-decoration-color: ${rgb(46, 50, 52)} !important;
}
```